sql/parser/eval.go

```diff
@@ -113,6 +180,32 @@ func (d DTuple) Type() string {
 	return "tuple"
 }
 
+// Compare implements the Datum interface.
+func (d DTuple) Compare(other Datum) int {
+	v, ok := other.(DTuple)
+	if !ok {
+		// Anything other than a DTuple compares greater (e.g. `(1, 2) > NULL`).
```

Comment:
giving `NULL` as the example here is just a bit misleading. I understand that's the case you're solving for, though.